src/maybe.ts

```diff
@@ -578,7 +580,8 @@ export function just<T>(value?: T | null): Maybe<T> {
   @returns     An instance of `Maybe.Nothing<T>`.
  */
 export function nothing<T>(_?: null): Maybe<T> {
-  return new Nothing<T>(_);
+  if (!NOTHING) NOTHING = new Nothing();
```

Comment:
Didn't your mother ever teach you to avoid double negatives?

Reply:
😂